examples/fetch-embed-job-example/10_fetch_confluence_space.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021-2024 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import csv
+import logging
+
+from langchain_community.document_loaders import ConfluenceLoader
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def fetch_confluence_space(url, token, space_key):
+    try:
+        # For more info regarding the LangChain ConfluenceLoader:
+        # https://python.langchain.com/docs/integrations/document_loaders/confluence
+        loader = ConfluenceLoader(url=url, token=token)
+        documents = loader.load(
+            space_key=space_key, include_attachments=True, limit=50, max_pages=50
+        )
+        return documents
+    except Exception as e:
+        log.error(f"Error fetching documents from Confluence: {e}")
+        return None
+
+
+def write_documents_to_csv(documents, filename):
+    with open(filename, mode="w", newline="", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        for doc in documents:
+            writer.writerow([doc.page_content])
+
+
+def run(job_input: IJobInput):
+    log.info(f"Starting job step {__name__}")
+
+    confluence_url = "https://yoansalambashev.atlassian.net/"
+    # since the Confluence space is public, no need to generate API token
+    token = ""
+    space_key = "RESEARCH"
+    output_csv = "documents.csv"
```

Comment:
I know that it's prototype/poc but some simple good coding practices apply and should be followed in prototypes (building good habits) 

It's better to create config.py or util.py and put common variables and configs there - e.g documnents_csv_file_location. which is reused in later step.

Reply:
addressed